java/src/jmri/script/JmriScriptEngineManager.java

```diff
@@ -202,6 +205,7 @@ public ScriptEngine getEngine(String engineName) {
                 // Setup the default python engine to use the JMRI python properties
                 this.initializePython();
             } else {
+                System.out.println("Create engine for "+engineName);
```

Comment:
Oops. Thanks for catching that.  Will fix.

Bob

> On Oct 18, 2015, at 4:05 AM, Randall Wood <notifications@github.com> wrote:
> 
> In java/src/jmri/script/JmriScriptEngineManager.java:
> 
> > @@ -202,6 +205,7 @@ public ScriptEngine getEngine(String engineName) {
> >                  // Setup the default python engine to use the JMRI python properties
> >                  this.initializePython();
> >              } else {
> > +                System.out.println("Create engine for "+engineName);
> 
> Did you really intend to push this to SYSOUT and not into a log?
> 
> —
> Reply to this email directly or view it on GitHub.
> 

--
Bob Jacobsen
jacobsen@mac.com +1-510-486-7355 AIM, Skype JacobsenRG